packages/arb-token-bridge-ui/src/components/TransferPanel/TokenRow.tsx

```diff
@@ -33,17 +32,18 @@ import { useAccountType } from '../../hooks/useAccountType'
 import { useNativeCurrency } from '../../hooks/useNativeCurrency'
 import { useNetworks } from '../../hooks/useNetworks'
 import { useNetworksRelationship } from '../../hooks/useNetworksRelationship'
+import { TokenLogoFallback } from './TokenInfo'
 
 function tokenListIdsToNames(ids: number[]): string {
   return ids
     .map((tokenListId: number) => listIdsToNames[tokenListId])
     .join(', ')
 }
 
-function TokenLogoFallback() {
+function StyledLoader() {
   return (
-    <div className="flex h-8 w-8 min-w-[2rem] items-center justify-center rounded-full bg-ocl-blue text-sm font-medium text-white">
-      ?
+    <div className="mr-2">
+      <Loader color="white" size="small" />
     </div>
   )
 }
```

Comment:
`TokenLogoFallback` is deleted because the `TokenRow` now uses `<TokenLogoFallback />` from `TokenInfo.tsx`